### PR TITLE
Windows build fix

### PR DESCRIPTION
**Pull request by modern-sapien**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: scripts/gulp/tasks/gulpGraphql.ts [hunk 1]
```diff
@@ -43,8 +43,9 @@ export async function graphqlCodegen () {
 }
 
 export async function graphqlCodegenWatch () {
-  const spawned = spawn('graphql-codegen', ['--watch', '--config', 'graphql-codegen.yml'], {
+  const spawned = spawn('bash.exe', ['graphql-codegen','--watch', '--config', 'graphql-codegen.yml'], {
```
Comment (elevatebart) [inline scripts/gulp/tasks/gulpGraphql.ts:46]: is this working on macos?

File: scripts/gulp/tasks/gulpWebpack.ts [hunk 2]
```diff
@@ -22,14 +22,16 @@ type RunWebpackCfg = {
 
 export async function runWebpack (cfg: RunWebpackCfg) {
   const { cwd, args = [], env = process.env, devServer = false, prefix } = cfg
+  console.log(cwd);
   const dfd = pDefer()
   const spawned = spawn(
-    devServer
+  `bash.exe ${devServer
```
Comment (elevatebart) [inline scripts/gulp/tasks/gulpWebpack.ts:28]: this is unnecessary, one can simply add the `.cmd` at the end to get the windows version

**Review by elevatebart — CHANGES_REQUESTED** [review]

**Comment by elevatebart** [comment]
I believe this PR is accomplishing the same goal. https://github.com/cypress-io/cypress/pull/19187

**Comment by jennifer-shehane** [comment]
@elevatebart Thanks bart. Just wanted to surface in case it was useful. Feel free to close.

**Pull request closed without merging**